hydrabus_framework/core/command/back.py

```diff
@@ -0,0 +1,11 @@
+from hydrabus_framework.modules.base import BaseModule
+
+
+def back(hbf_instance):
+    """
+    Move back from the current context
+    :param hbf_instance: Hydrabus framework instance (self)
+    :return:
+    """
+    hbf_instance.current_module = BaseModule()
+    hbf_instance.update_prompt("")
```

Comment:
This will not work if I use toto then tata. When calling back, I will not be redirected to toto

Reply:
Still wrong. Back seems that you will switch to the previous state. By setting the current_module to None, you don't rollback to the prior state.

Reply:
Done [#a10b774](https://github.com/hydrabus-framework/framework/commit/a10b774fbb75b0f4bb33caa46e07f297efc17acc)